src/ray/common/test/testing.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains a few util macros for testing.
+
+#pragma once
+
+#define RAY_EXPECT_OK(s) EXPECT_TRUE((s).ok())
+
+#define RAY_ASSERT_OK(s) ASSERT_TRUE((s).ok())
```

Comment:
I'm actually not sure what values these two macros bring. Directly doing `EXPECT_TRUE((s).ok())` seems simple enough.

Reply:
It saves typing, otherwise why all status class define a `ok` method, i.e. https://github.com/abseil/abseil-cpp/blob/f339ea31b8dfc139075ba3feb4d848beaee5639e/absl/status/status.h#L477-L482

Reply:
ok 

```
>>> len("RAY_EXPECT_OK(s)")
16
>>> len("EXPECT_TRUE(s.ok())")
19
```

save 3 characters each time.

Reply:
Another reason, gtest allows stream operation at expectation and assertion
```
EXPECT_TRUE(...) << ...;
ASSERT_TRUR(...) << ...;
```

But it requires all possible T for `StatusOr<T>` to be printable; which is exactly the universal printer I mentioned earlier if you remember. Need some time to implement, not a short PR.